programs/nft-club/src/components/creator.rs

```diff
@@ -70,6 +93,24 @@ pub struct DeleteAccount<'info> {
     pub system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+#[instruction(username: String, email: String, description: String)]
```

Comment:
not important for this PR, but is there a way we could set this up so that I can just update a single value on the account and not have to pass along values that I don't intend to change?